docs/supported-binary-in-carbondata.md

```diff
@@ -0,0 +1,162 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more 
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership. 
+    The ASF licenses this file to you under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with 
+    the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software 
+    distributed under the License is distributed on an "AS IS" BASIS, 
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and 
+    limitations under the License.
+-->
+
+# Support binary data type in CarbonData
```

Comment:
Binary datatype is a standard datatype which we are supporting now. So better no need to add any special information about this datatype.  Just add along with other datatypes in the document.
And regarding example, you can add one more column with binary  in the existing example but not to the document

Reply:
ok, remove